python/alibiexplainer/Makefile

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Should this makefile live at the python level and have targets for all python packages we have?